app/src/main/java/com/fibelatti/raffler/views/activities/GroupActivity.java

```diff
@@ -250,4 +258,22 @@ private void startSubGroupsActivity(Group group) {
         intent.putExtra(Constants.INTENT_EXTRA_GROUP, group);
         startActivity(intent);
     }
+
+    private void shareGroup(Group group) {
+        FileHelper fileHelper = new FileHelper(context);
+        Intent shareIntent = new Intent();
+
+        if (fileHelper.createFileFromGroup(group)) {
+            Uri uri = FileProvider.getUriForFile(context,
+                    getString(R.string.file_provider_authority),
+                    new File(fileHelper.getGroupFilePath()));
+
+            shareIntent.setAction(Intent.ACTION_SEND);
```

Comment:
@fibelatti could create one method for this intent. And sending the uri like a parameter